src/components/cylc/table/Table.vue

```diff
@@ -243,42 +249,42 @@ export default {
         },
         {
           text: 'Cycle Point',
-          value: 'node.cyclePoint',
+          value: 'node_cyclePoint',
           sort: (a, b) => DEFAULT_COMPARATOR(String(a ?? ''), String(b ?? ''))
         },
         {
-          text: 'Host',
-          value: 'latestJob.platform',
+          text: 'Platform',
+          value: 'latestJob_platform',
           sort: (a, b) => DEFAULT_COMPARATOR(a ?? '', b ?? '')
         },
         {
-          text: 'Job System',
+          text: 'Job Runner',
           value: 'latestJob.jobRunnerName',
           sort: (a, b) => DEFAULT_COMPARATOR(a ?? '', b ?? '')
         },
         {
           text: 'Job ID',
-          value: 'latestJob.jobId',
+          value: 'latestJob_jobId',
           sort: (a, b) => parseInt(a ?? 0) - parseInt(b ?? 0)
         },
         {
-          text: 'T-submit',
-          value: 'latestJob.submittedTime',
+          text: 'T-Submit',
+          value: 'latestJob_submittedTime',
           sort: datetimeComparator
         },
         {
-          text: 'T-start',
-          value: 'latestJob.startedTime',
+          text: 'T-Start',
+          value: 'latestJob_startedTime',
           sort: datetimeComparator
         },
         {
-          text: 'T-finish',
-          value: 'latestJob.finishedTime',
+          text: 'T-Finish',
+          value: 'latestJob_finishedTime',
           sort: datetimeComparator
         },
         {
-          text: 'dT-mean',
-          value: 'meanElapsedTime',
+          text: '&Delta;T-Mean',
```

Comment:
`&Delta;` doesn't get expanded into a delta symbol. If you wanted the unicode delta symbol here, I think you can write it in plain text, no escape needed, the web runs on UTF-8.

Reply:
BTW this did render a Delta symbol - see the screenshot on this issue. The reason was the `v-html` template you commented on above https://github.com/cylc/cylc-ui/pull/1112/files#r1045695288 ... However, you're right about just using UTF-8!